src/circle.tsx

```diff
@@ -1,14 +1,17 @@
 import * as React from 'react';
-import {Component} from 'react';
+import { Component } from 'react';
 
 export interface CircleProps {
   progress: number;
   showPercentage?: boolean;
+  showPercentageSymbol: boolean;
   progressColor?: string;
   bgColor?: string;
   textColor?: string;
   size?: string;
   lineWidth?: string;
+  percentSpacing: number;
+  textStyle?: object;
```

Comment:
I think we can find a better typing for this, I believe **CSSProperties** is the type we want. You can import it from the react package ```import {CSSProperties} from 'react';```